enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/EnterpriseCoreEditionModule.java

```diff
@@ -193,6 +200,15 @@ public void registerEditionSpecificProcedures( Procedures procedures ) throws Ke
         life.add( coreServerModule.membershipWaiterLifecycle );
     }
 
+    static Predicate<String> fileWatcherFileNameFilter()
+    {
+        return Predicates.all(
+                fileName -> !fileName.startsWith( PhysicalLogFile.DEFAULT_NAME ),
+                filename -> !filename.startsWith( StoreUtil.BRANCH_SUBDIRECTORY ),
```

Comment:
Yes branched is impossible, but we still use temp-copy. See TemporaryStoreDirectory (which has it hardcoded but the same as the one in StoreUtil).